osu.Game/Skinning/ISerialisableDrawableContainer.cs

```diff
@@ -14,11 +14,32 @@ namespace osu.Game.Skinning
     /// </summary>
     public interface ISerialisableDrawableContainer : IDrawable
     {
+        /// <summary>
+        /// The lookup criteria which will be used to retrieve components from the active skin.
+        /// </summary>
+        ISkinComponentLookup Lookup { get; }
+
         /// <summary>
         /// A bindable list of components which are being tracked by this skinnable target.
         /// </summary>
         IBindableList<ISerialisableDrawable> Components { get; }
 
+        /// <summary>
+        /// Whether all components in this container have been loaded.
+        /// </summary>
+        bool ComponentsLoaded { get; }
+
+        /// <summary>
+        /// Whether the components in this container can be edited by an end user.
+        /// </summary>
+        bool IsEditable => true;
+
+        /// <summary>
+        /// Whether the components in this container are static.
+        /// If <c>true</c>, components may not be added or removed from this container.
+        /// </summary>
+        bool IsStatic => false;
```

Comment:
Why do these exist simultaneously? I'd expect `!IsEditable` implies `IsStatic`.

Reply:
Hmm. I'd probably rename `IsStatic` to `ContentsMutable` or something, because "static" feels ambiguous to me (static in what respect?)